README.md

```diff
@@ -188,113 +252,166 @@ The following CSS:
 }
 ```
 
-will be processed to:
+This will be transformed to:
 
 ```css
 .box-foo {
 	width: 150px;
+}
 
-	.box-bar {
-		width: 150px;
-	}
-
-	@media (max-width: 800px) {
-		.box-bar {
-			width: 300px;
-		}
-	}
+.box-foo .box-bar {
+	width: 150px;
 }
 
 @media (max-width: 800px) {
 	.box-foo {
 		width: 300px;
 	}
+	
+	.box-foo .box-bar {
+		width: 300px;
+	}
 }
 ```
 
+### `calc()`
```

Comment:
Should postcss-calc still be mentioned, though? I do feel it's still a good idea to mention at least.